### ISSUE TITLE
training loss is not reducing

### ISSUE THREAD

**Issue by tomwuu** [facebookresearch/maskrcnn-benchmark#1210]
cause I only have one GPU,  I set
SOLVER:
  BASE_LR: 0.0025
  WEIGHT_DECAY: 0.0001
  STEPS: [0, 30000, 40000]
  MAX_ITER: 60000
  IMS_PER_BATCH: 2

and training loss almost is not reducing and shake extremely ｂａｄｌｙ, so it cannot get good result. the model is not trained right.

<img width="766" alt="WX00020113-181853@2x" src="https://user-images.githubusercontent.com/35363538/72244334-67bda280-3631-11ea-9568-8074a1c8ca83.png">

**Comment by sno6** [comment]
@tomwuu Hello, how did you end up solving this?

**Comment by tomwuu** [comment]
At first you should set a right learning rate, 
And the pre-trained 50-ResNet weight provided is wrong, you better download yourself.
And finally the loss actually would not be apparently down soon,  it may start from 1.X to 0.5 when you finish your training.

**Comment by sno6** [comment]
@tomwuu Thanks for the info :)
